runtimes/rocm/rocm.zig

```diff
@@ -14,6 +19,44 @@ fn hasRocmDevices() bool {
     return true;
 }
 
+fn setupRocmEnv() !void {
```

Comment:
same here, it's better to keep `load` which is the main entry point at the top of the file